spring-cloud-kubernetes-config/src/main/java/org/springframework/cloud/kubernetes/config/reload/ConfigReloadAutoConfiguration.java

```diff
@@ -48,7 +48,8 @@
  * @author Nicolla Ferraro
  */
 @Configuration(proxyBeanMethods = false)
-@ConditionalOnProperty(value = "spring.cloud.kubernetes.enabled", matchIfMissing = true)
+@ConditionalOnProperty(name = "spring.cloud.kubernetes.config.enabled",
```

Comment:
This probably needs both right? If the whole thing is disabled or config is disabled.

Reply:
Although a test triage didn't active config, but agree, it would be clear to have both.